x-pack/plugins/maps/public/routing/routes/maps_app/index.ts

```diff
@@ -50,9 +59,16 @@ function mapStateToProps(state = {}) {
   };
 }
 
-function mapDispatchToProps(dispatch) {
+interface DispatchSetQueryArgs {
+  filters?: Filter[];
+  query?: Query;
+  timeFilters?: TimeRange;
+  forceRefresh?: boolean;
+}
+
+function mapDispatchToProps(dispatch: ThunkDispatch<Promise<string>, MapStoreState, any>) {
   return {
-    dispatchSetQuery: ({ forceRefresh, filters, query, timeFilters }) => {
+    dispatchSetQuery: ({ forceRefresh, filters, query, timeFilters }: DispatchSetQueryArgs) => {
```

Comment:
Since `DispatchSetQueryArgs` are only used here, can you just inline the interface vs giving it a name?

Reply:
Fixed this.